e2e-tests/src/test/java/com/huawei/cloud/tests/OtcTransferEndToEndTest.java

```diff
@@ -103,33 +123,66 @@ void obsToObsTransfer() {
         var f = getFileFromResourceName(TESTFILE_NAME);
         providerClient.putObject(sourceBucket, TESTFILE_NAME, f);
 
-        createResourcesOnProvider(assetId, sourceAddress(sourceBucket, prefix));
+        JsonArrayBuilder allowedSourceTypes = Json.createArrayBuilder();
+        allowedSourceTypes.add(Json.createValue(ObsBucketSchema.TYPE));
+        JsonArrayBuilder allowedDestTypes = Json.createArrayBuilder();
+        allowedDestTypes.add(Json.createValue(ObsBucketSchema.TYPE));
+        JsonArrayBuilder allowedTransferTypes = Json.createArrayBuilder();
+        allowedTransferTypes.add(Json.createValue(ObsBucketSchema.TRANSFERTYPE_PUSH));
 
-        var transferProcessId = CONSUMER.requestAsset(PROVIDER, assetId, noPrivateProperty(), obsSink(destBucket, prefix));
+        JsonObject dataPlaneRequestBody = Json.createObjectBuilder().add("@context", Json.createObjectBuilder().add("@vocab", "https://w3id.org/edc/v0.0.1/ns/"))
+                .add("@id", "http-push-provider-dataplane")
+                .add("url", PROVIDER.getControlEndpoint().getUrl().toString().concat("/transfer"))
+                .add("allowedSourceTypes", allowedSourceTypes.build())
+                .add("allowedDestTypes", allowedDestTypes.build())
+                .add("allowedTransferTypes", allowedTransferTypes)
+                .add("properties", "").build();
 
+        PROVIDER.getManagementEndpoint().baseRequest().contentType(ContentType.JSON).body(dataPlaneRequestBody).when().post("/v2/dataplanes");
+        createResourcesOnProvider(assetId, sourceAddress(sourceBucket, prefix));
+
+        var transferType = ObsBucketSchema.TRANSFERTYPE_PUSH;
+        var offer = getOfferForAsset(PROVIDER, assetId);
+        var contractAggId = CONSUMER.negotiateContract(PROVIDER, offer);
+        var transferProcessId = CONSUMER.initiateTransfer(PROVIDER, contractAggId, null, obsSink(destBucket, prefix), transferType);
         await().atMost(TIMEOUT).untilAsserted(() -> {
             var state = CONSUMER.getTransferProcessState(transferProcessId);
-            assertThat(state).isEqualTo(COMPLETED.name());
+            assertThat(TransferProcessStates.valueOf(state).code()).isGreaterThanOrEqualTo(STARTED.code());
```

Comment:
Test needs to wait for the TransferProcess to be completed.
So this Test is not valid